### PR TITLE
feat(transform): add SimpleBrightness

### PR DESCRIPTION
**Pull request by nikitavbv**

### CODE REVIEW

File: src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java [hunk 1]
```diff
@@ -30,6 +30,29 @@ public byte blue() {
     return blue;
   }
 
+  public Pixel brighter(double factor) {
+    int i = (int)(1.0/(1.0-factor));
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java]: Maybe rename to something like intensity to better understand what is doing

Comment (nikitavbv) [reply src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java]: Done.

File: src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java [hunk 2]
```diff
@@ -30,6 +30,29 @@ public byte blue() {
...
+  public Pixel brighter(double factor) {
+    int intensity = (int)(1.0/(1.0-factor));
+    if ( red == 0 && green == 0 && blue == 0) {
+      return new Pixel((byte) intensity, (byte) intensity, (byte) intensity);
+    }
+    int r = red & 0xFF;
+    int g = green & 0xFF;
+    int b = blue & 0xFF;
+    if (r > 0 && r < intensity) {
+      r = i;
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java]: You have forgotten to rename i here

File: src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java [hunk 3]
```diff
@@ -30,6 +30,29 @@ public byte blue() {
...
+    }
+    if (g > 0 && g < intensity) {
+      g = i;
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java]: Same as above

File: src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java [hunk 4]
```diff
@@ -30,6 +30,29 @@ public byte blue() {
...
+    }
+    if (b > 0 && b < intensity) {
+      b = i;
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java]: Same as above

**Review by Brookoko — CHANGES_REQUESTED** [review]

**Review by Brookoko — APPROVED** [review]

**Pull request merged**